src/flask/config.py

```diff
@@ -245,7 +246,7 @@ def from_file(
             app.config.from_file("config.json", load=json.load)
 
             import toml
-            app.config.from_file("config.toml", load=toml.load)
+            app.config.from_file("config.toml", load=toml.load, text=True)
```

Comment:
This still uses the `toml` library, it needs to change to `tomllib` to be a useful example of the parameter.